hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/UpdateProcessorTest.java

```diff
@@ -140,6 +143,50 @@ public void when_serializedObject_then_deserializedCorrect() {
                 .isEqualTo(processor);
     }
 
+    @Test
+    public void when_manualUpdatingEntryProcessorWithGetDdlExpression_then_success() throws ExecutionException, InterruptedException {
```

Comment:
OS version does not provide any security, so hacking it is like beating a dead horse. There is no requirement that `get_ddl` with tampered subject should work in OS - it is "don't care" in case of OS.

I think that tests in EE are sufficient and this one can be removed.